python/foglamp/services/core/server.py

```diff
@@ -654,9 +654,13 @@ def stop(cls, loop=None):
             found_services = ServiceRegistry.get()
             services_to_stop = list()
 
+            from foglamp.common.service_record import ServiceRecord
```

Comment:
No! As discussed earlier it must be very very uncommon case. Let's not make it a practice. Figure out and refactor to remove it

Reply:
Line moved to be with other import statements. Is this you meant?